extension/src/cli/version.ts

```diff
@@ -25,12 +29,12 @@ const getTextAndSend = (version: string, update: 'CLI' | 'extension'): void => {
 
 const warnIfMinorAhead = (
   currentMajor: number,
-  minMajor: number,
+  buildMajor: number,
   currentMinor: number,
-  minMinor: number
+  buildMinor: number
 ) => {
-  if (currentMajor === minMajor && currentMinor > minMinor) {
-    Toast.warnWithOptions(`The located version of the CLI is at least a minor version ahead of the expected version. 
+  if (currentMajor === buildMajor && currentMinor > buildMinor) {
+    Toast.warnWithOptions(`The located version of the CLI is at least a minor version ahead of the latest version this version of extension was tested with. 
```

Comment:
[C] Too many `versions` in this sentence. 

How about 
```
The located CLI is at least a minor version ahead of the latest version the extension was tested with.
```

Reply:
done, thanks!